swan-lake/development-tutorials/source-code-dependencies/style-guide/annotations-documentation-and-comments.md

```diff
@@ -41,56 +41,34 @@ function testFunction() {
 **Example,**
   
 ```ballerina
+// Declares an annotation tag on the type.
+annotation record {||} validated on parameter, return;
+
 // Parameter annotation.
-public function secureFunction1(@untainted string secureInName, @untainted int secureInId, string insecureIn) {
+public function secureFunction1(@validated string secureInName, int secureInId, string insecureIn) {
     ...
 }
-  
-public function secureFunction2(@untainted string secureInName,
-    @untainted int secureInId, string insecureIn) {
+
+public function secureFunction2(string secureInName,
+    @validated int secureInId, string insecureIn) {
     ...
 }
-  
+
 // Return type annotation.
-public function taintedReturn1() returns @tainted string {
-    ...
+<<<<<<< HEAD
```

Comment:
Conflicts not resolved correctly 
<<<<<<< HEAD
>>>>>>> 196f3d1e6e9b8c091c32f2ae2d92d45ca9002d09

should not appear after resolving

Reply:
Fixed.